larq_compute_engine/core/types.h

```diff
@@ -1,9 +1,18 @@
 #ifndef COMPUTE_ENGINE_CORE_TYPES_H_
 #define COMPUTE_ENGINE_CORE_TYPES_H_
 
+#include <cstdint>
+#include <limits>
+#include <type_traits>
+
 namespace compute_engine {
 namespace core {
 
+// Define these once here, so they can be included everywhere.
+using TBitpacked = std::int32_t;
+constexpr std::size_t bitpacking_bitwidth =
+    std::numeric_limits<typename std::make_unsigned<TBitpacked>::type>::digits;
+
```

Comment:
Just to highlight for reviewers, this is the most important addition, where `TBitpacked` and `bitpacking_bitwidth` are defined.